device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/qos.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json.j2
```

Comment:
pg_profile is pointing to ../Mellanox-SN3800-D112C8/pg_profile_lookup.ini. Can you confirm if these soft-links are correct?

Reply:
yes I verified it on the workspace, the symbolic link is there. I also verified the pg_profile_lookup.ini file is viewable on the switch as well.